pydantic/types.py

```diff
@@ -436,6 +437,12 @@ def constr(
     return _registered(type('ConstrainedStrValue', (ConstrainedStr,), namespace))
 
 
+class StrLimited:
```

Comment:
you might want to implement `__get_validators__` and raise an error people try to use `StrLimited` with parameterising it.